learn/advanced/updating.md

```diff
@@ -201,59 +147,228 @@ The server should return a response that looks like this:
+
+Once Meilisearch v1 is released, this two-step process won't be necessary as v1 will be compatible with dumps from all previous versions.
+:::
+
+## Step 3: Import data
+
+### Launch Meilisearch and import the dump
+
+Execute the command below to import the dump at launch:
+
+:::: tabs
+
+::: tab Local installation
+
+```bash
+# replace {dump_uid.dump} with the name of your dump file
+./meilisearch --import-dump dumps/{dump_uid.dump} --master-key="MASTER_KEY"
+```
+
+:::
+
+::: tab Cloud platforms
+
+```sh
+# replace {dump_uid.dump} with the name of your dump file
+meilisearch --db-path /var/lib/meilisearch/data.ms --import-dump "/var/opt/meilisearch/dumps/{dump_uid.dump}"
+```
+
 :::
 
+::::
+
 Importing a dump requires indexing all the documents it contains. Depending on the size of your dataset, this process can take a long time and cause a spike in memory usage.
 
-Finally, don’t forget to set `displayedAttributes` back to its previous value if necessary. You can do this using the [update displayed attributes endpoint](/reference/api/settings.md#update-displayed-attributes).
+### Restart Meilisearch as a service
+
+If you're running a **cloud instance**, press `Ctrl`+`C`  to stop Meilisearch once your dump has been correctly imported. Next, execute the following command to run the script to configure Meilisearch and restart it as a service:
+
+```
+meilisearch-setup
+```
 
-Congratulations! You have successfully migrated your Meilisearch database to the latest version! 🎉
+If required, set `displayedAttributes` back to its previous value using the [update displayed attributes endpoint](/reference/api/settings.md#update-displayed-attributes).
+
+## Conclusion
+
+Now that your updated Meilisearch instance is up and running, verify that the dump import was successful and no data was lost.
+
+If everything looks good, then congratulations! You successfully migrated your database to the latest version of Meilisearch. Be sure to check out the [changelogs](https://github.com/meilisearch/MeiliSearch/releases).
+
+If something went wrong, you can always roll back to the previous version and try again. Be sure to check out the [version-specific update instructions](#version-specific-update-instructions), and feel free to [reach out for help](https://discord.gg/meilisearch) if the problem continues.
+
+### Delete backup files or rollback (_optional_)
+
+Delete the Meilisearch binary and `data.ms` folder created in the previous step. Next, move the old files back to their previous location using:
+
+:::: tabs
+
+::: tab Local installation
+
+```
+mv /tmp/meilisearch /path/to/your/meilisearch/directory/meilisearch
+mv /tmp/data.ms /path/to/your/meilisearch/directory/data.ms
```

Comment:
AFAIK for default meilisearch (no `--db-path` given), `data.ms` **must be found within the Meilisearch binary directory**. In which case, if users executed the code snippet you wrote, Meilisearch would not be able to start.

```suggestion
mv /tmp/data.ms /path/to/your/meilisearch/directory/meilisearch/data.ms
```

LMK if you think I'm wrong about this.